diam/avp/codes.go

```diff
@@ -59,28 +53,43 @@ const (
 	AoCServiceObligatoryType              = 2312
 	AoCServiceType                        = 2313
 	AoCSubscriptionInformation            = 2314
-	ApplicID                              = 1218
 	ApplicationPortIdentifer              = 3010
 	ApplicationProvidedCalledPartyAddress = 837
 	ApplicationServer                     = 836
 	ApplicationServerID                   = 2101
 	ApplicationServerInformation          = 850
 	ApplicationServiceProviderIdentity    = 532
 	ApplicationSessionID                  = 2103
+	ApplicID                              = 1218
```

Comment:
This is actually from TGPP spec, name is indeed confusing, but it is there (TS 32.299 ). It looks modified since I have run autogen.sh script after adding application types to dictionaries. But it was there since TGPP specific avps have been added.